plugins/orchestrator-backend/src/service/api/mapping/V2Mappings.ts

```diff
@@ -0,0 +1,215 @@
+import moment from 'moment';
+
+import {
+  ASSESSMENT_WORKFLOW_TYPE,
+  ExecuteWorkflowResponseDTO,
+  ProcessInstance,
+  ProcessInstanceDTO,
+  ProcessInstanceState,
+  ProcessInstanceStatusDTO,
+  WorkflowCategory,
+  WorkflowCategoryDTO,
+  WorkflowDataDTO,
+  WorkflowDefinition,
+  WorkflowDTO,
+  WorkflowExecutionResponse,
+  WorkflowListResult,
+  WorkflowListResultDTO,
+  WorkflowOverview,
+  WorkflowOverviewDTO,
+  WorkflowSpecFile,
+  WorkflowSpecFileDTO,
+} from '@janus-idp/backstage-plugin-orchestrator-common';
+
+// Mapping functions
+export function mapToWorkflowOverviewDTO(
+  overview: WorkflowOverview,
+): WorkflowOverviewDTO {
+  return {
+    workflowId: overview.workflowId,
```

Comment:
you can use object spread to make this shorter
return {
  ...overview,
  category: mapWorkflowCategoryDTOFromString(overview.category),
}
same in other mapping functions

Reply:
I agree that it looks nicer, but if it's ok for you, I'd prefer to do these changes in a following PR to be sure I won't break anything (I'll add more unit tests).